test/convertors/expression-convertor.test.ts

```diff
@@ -1,15 +1,37 @@
 import { ExpressionConvertor } from "../../src/convertors/expression-convertor";
 import { ExpressionTreeNode } from "../../src/models/tree-node";
 
+const ATOMIC_EXPRESSIONS: string[][] = [
+    ["1 + 2"],
+    ["1 - 2"],
+    ["1 * 2"],
+    ["1 / 2"],
+    ["1 ** 2"],
+    ["sqrt(1)"]
+]
+
+const COMPOUND_EXPRESSIONS: string[][] = [
```

Comment:
Let's add some expressions that contain also subtraction, division and square root